mksproxy_version5.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
```

Comment:
These files can be combined. And running them takes a required param that includes the version for the proxy, which decides which handler class to use.